### ISSUE TITLE
IDEATION: Widespread assumption and usage re ideas getting discussed further in the discussion phase

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1370]
**Insights from Round 2 ideation UX testing:**  

- Something that's come out quite clearly is that people are generally assuming that the discussions phase is for discussing the ideas, or certain ideas. 
- Some people are just very confidently assuming this (when getting their bearings on arrival into the topic), and some are more hesitant - "But then you have the replies on ideas for discussion so I'm not sure". 
- Admins are often trying to set up the discussion phase to use it as a place for discussing the pros and cons of certain ideas. They therefore ask for multiple discussion questions, as currently they can only discuss one idea.  
- People initially (again, when getting their bearing in the topic) say things along the lines of "Sending the ideas to discussion", seemingly assuming that some of the ideas can be moved to the discussion phase. 
- There is a bit of confusion in how users are referring to the idea gathering phase and the discussion, e.g. calling everything that happened prior to voting (both ideas and discussion) "the discussion", or saying "I will open up a discussion on X", meaning I will add an idea about X. 
- Will also note here that admins are often assuming that discussion and idea gathering can be open at the same time (perhaps due to thinking the discussion part is for discussing ideas), but I will make a separate issue about this.

**Suggestions:**

I imagine a lot of this is for the 'bigger fixes' work, but as a pre-ideation launch fix, @ilmartyrk is it technically difficult/time-consuming to add multiple discussions already? I'm thinking that if having multiple discussion widgets is quite simple (and if @kevincrepin could design it quite quickly) then we could already add these as part of the ideation launch, and just leave the multiple votes for later?

**Comment by ilmartyrk** [comment]
@BeccaMelhuish @kevincrepin what if we allow pro con discussion under ideas instead of replies?

**Comment by BeccaMelhuish** [comment]
@ilmartyrk @kevincrepin It did occur to me while watching people respond to ideas that this could be useful (as most of the replies were either clearly pro or con - more so than contributions in discussions, which don't always fit those categories well). 

But it feels like a bit of a big change to make without another round of testing.. Might potentially cause more confusion re what the discussion space is for, plus would need more stats about pros and cons replies to each idea to show up somewhere (like we do for discussions), etc. 

I did ask a few users if they would want pros and cons on idea replies, and they said they didn't see the need. And none of the others suggested it. But then they were often thinking that would happen in discussion, which it currently can't, so isn't certain they wouldn't want it while we don't have multiple discussion widgets.. 

From what I gathered, they quite liked the flow of gather ideas first, then discuss the pros and cons of some of them separately in the discussion phase. But not certain. 

I'm thinking this could be very useful though in cases where there are lots of replies on an idea. Then rather than admins having to read every reply to see if the feeling in the reply is positive or negative, they would be able to see a summary of how many people post pro replies and con replies. One admin did mention a frustration at not being able to understand quickly how the likes and dislikes on the idea replies relate to the overall sentiment on the idea itself. So this solution would fit well with solving that. 

So in summary: I think would be a good thing to consider as part of the bigger fixes, and test it out a bit, but for the ideation launch I think not (unless we do another quick testing round..) 

@ilmartyrk is it quite a big job to add multiple discussion widgets? Though I guess that's also quite a big change to add without further testing..

**Comment by BeccaMelhuish** [comment]
One possible solution as a quick fix for now, would be to just make it clearer in the copy that the discussion phase can only have one discussion question (for now). At least then admins wouldn't find themselves in the situation of having started a discussion on one idea only, then finding they can't do the same for the rest, but have already closed the ideation where further discussion on the others could have happened. In which case, they would most likely just skip the discussion phase, or discuss another related issue.

**Comment by kevincrepin** [comment]
I've been working on this for a bit now and I think I would also vote for including this in the bigger fix (the one where each phase gets its own text).

It becomes quite complex when adding discussion phase after ideation and at that point allowing the admin to create the discussion phase immediately with several discussion prompts, which could be either custom or based on ideas. Since all of this again has to happen within these dialog windows, it makes it really cramped and confusing.

**Comment by BeccaMelhuish** [comment]
@kevincrepin OK yes makes sense, let's leave it for bigger fixes then! 

What are your thoughts on these copy changes as short-term fixes though: 

- Discussion phase text: 

_In the discussion phase, topic admins can post one discussion question, on a matter related to the topic or ideas._ 

_Participants can add pro or con arguments in response to this question, and other participants can reply or react to these. The general sentiment of the discussion is shown visually._

_There is currently no discussion phase for this topic, but topic admins can start one once the idea gathering ends._

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/fb7cc4ee-25e2-4291-9b2b-b8f2e21563e5)

- Guidance note for discussion question fields:

_Please add a specific discussion question to bring focus for the participants. You can only post one discussion question per topic. Participants can respond with pro, con or informational arguments._  

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/af404dda-f033-4c65-aff0-6b920b5dc5fa)

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/1e5cb299-1fbc-4b4d-ad68-4c1703f5160e)

- Start discussion phase text edit:

_"have a pro-con discussion on something,"_ 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/2845f1e9-bffc-4b53-8b71-a8517db6dfdb)

**Comment by kevincrepin** [comment]
Those suggestions seem good to me!

**Comment by BeccaMelhuish** [comment]
@ilmartyrk, Added 'high priority' but just for the text changes

**Comment by ilmartyrk** [comment]
est. dev 5mins

**Comment by BeccaMelhuish** [comment]
Checked texts all looks good, moving to done :)
